<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UC_Factories</class>
 <widget class="QFrame" name="UC_Factories">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1260</width>
    <height>860</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QFrame" name="Factory_Info">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1260</width>
     <height>860</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="Factory_Properties">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1031</width>
      <height>860</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="btn_add_msn3">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>770</y>
       <width>101</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>إضافة</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_edit_msn3">
     <property name="geometry">
      <rect>
       <x>285</x>
       <y>770</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>تعديل</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_79">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>560</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string>إسم المالك</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_276">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>270</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string>البلوك</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_77">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>350</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string>المساحة (متر مربع)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_remove_msn3">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>770</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>حذف</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_78">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>420</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string>السعر</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="txtDetails">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>481</width>
       <height>121</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="placeholderText">
      <string> تفاصيل اخرى عن المصنع</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtArea">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>340</y>
       <width>330</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_76">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>70</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string>القطعه</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtPieceID">
     <property name="geometry">
      <rect>
       <x>589</x>
       <y>70</y>
       <width>271</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtOwnerPhone">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>480</y>
       <width>330</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_275">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>200</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string>الحى</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtPrice">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>410</y>
       <width>330</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>340</y>
       <width>481</width>
       <height>411</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="title">
      <string>صور المصنع</string>
     </property>
     <widget class="QPushButton" name="btn_add_img">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>360</y>
        <width>35</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>simplified arabic</family>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_remove_img">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>360</y>
        <width>35</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>simplified arabic</family>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tblImgs">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>481</width>
        <height>381</height>
       </rect>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <column>
       <property name="text">
        <string>الصورة</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="btn_show_imgs">
      <property name="geometry">
       <rect>
        <x>324</x>
        <y>360</y>
        <width>61</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>simplified arabic</family>
        <pointsize>8</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>عرض</string>
      </property>
     </widget>
     <zorder>tblImgs</zorder>
     <zorder>btn_add_img</zorder>
     <zorder>btn_remove_img</zorder>
     <zorder>btn_show_imgs</zorder>
    </widget>
    <widget class="QLabel" name="label_80">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>490</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string>رقم المالك</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_314">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>700</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string>باقي الاقساط</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbxStatus">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>620</y>
       <width>331</width>
       <height>49</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>خالص الثمن </string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>باقي قسط</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="txtOwnerName">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>550</y>
       <width>330</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_315">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>630</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string>حالة العقار</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtRest">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>690</y>
       <width>331</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_277">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>140</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string>المدينه</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbxFactoryType">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>760</y>
       <width>330</width>
       <height>49</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>مبني متعدد الادوار</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>جمالون</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_84">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>770</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string> نوع المصنع</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbxCity">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>130</y>
       <width>330</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="cbxBlock">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>270</y>
       <width>330</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="cbxStreet">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>200</y>
       <width>330</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="txtGamalonDetails">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>481</width>
       <height>121</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="placeholderText">
      <string> تفاصيل اخرى عن المصنع</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSearch">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>70</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>imgs/search.png</normaloff>imgs/search.png</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="Factory_Archiev">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>0</width>
      <height>860</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QComboBox" name="comboBox_34">
     <property name="geometry">
      <rect>
       <x>796</x>
       <y>30</y>
       <width>150</width>
       <height>39</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_32">
     <property name="geometry">
      <rect>
       <x>276</x>
       <y>30</y>
       <width>150</width>
       <height>39</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_114">
     <property name="geometry">
      <rect>
       <x>950</x>
       <y>40</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string>المدينه</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_33">
     <property name="geometry">
      <rect>
       <x>536</x>
       <y>30</y>
       <width>150</width>
       <height>39</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_search_12">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>101</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>بحث</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_search_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>101</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>طباعه</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget_17">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>89</y>
       <width>1031</width>
       <height>760</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <column>
      <property name="text">
       <string>المدينه</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>الحى </string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>البلوك</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>رقم القطعه </string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>نوع المصنع</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>السعر</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>المساحه</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="label_115">
     <property name="geometry">
      <rect>
       <x>446</x>
       <y>40</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string>البلوك</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_116">
     <property name="geometry">
      <rect>
       <x>696</x>
       <y>40</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string>الحى</string>
     </property>
    </widget>
   </widget>
   <widget class="QListWidget" name="factory_list">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>0</y>
      <width>231</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 24px;</string>
    </property>
    <item>
     <property name="text">
      <string>  المصانع</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>  أرشيف المصانع</string>
     </property>
    </item>
   </widget>
   <zorder>Factory_Archiev</zorder>
   <zorder>Factory_Properties</zorder>
   <zorder>factory_list</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
